### PR TITLE
added tests to actorservice class

### PR DESCRIPTION
**Pull request by fribergjesper94**
Please take a look at my tests

### CODE REVIEW

File: src/test/java/com/example/almgroupproject/service/ActorServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+@ExtendWith(MockitoExtension.class)
+public  class ActorServiceTest {
+
+    @Mock
+    ActorRepository mockRepository;
+
+    @DisplayName("Gets all actors in the list")
+    @Test
+    public void getAllActorsTest () {
+
```
Comment (Elliotkongu) [inline src/test/java/com/example/almgroupproject/service/ActorServiceTest.java]: Onödig whitespace

File: src/test/java/com/example/almgroupproject/service/ActorServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    @Test
+    public void saveActorTest() {
+        Actor actor = new Actor(1, "Förnamn", "Efternamn", LocalDate.of(1985,8,20));
+        ActorService service = new ActorService(mockRepository);
+        assertEquals(service.save(actor), "Actor saved");
+        when(mockRepository.findActorByFirstNameAndLastName(
+                actor.getFirstName(),
+                actor.getLastName()))
+                .thenReturn(actor);
+                assertEquals(service.save(actor), "Actor already exists");
```
Comment (Elliotkongu) [inline src/test/java/com/example/almgroupproject/service/ActorServiceTest.java]: assertEquals() ska inte vara så långt tabbad

**Review by Elliotkongu — CHANGES_REQUESTED** [review]
småfixar behövs

**Review by Elliotkongu — APPROVED** [review]
Nu ser det snyggt ut

**Comment by UlfNyberg** [comment]
Please change id from int to String

**Comment by UlfNyberg** [comment]
you beat me to my comment

**Pull request merged**